ably/README.md

```diff
@@ -0,0 +1,64 @@
+## Overview
+The [Ably][1] platform is used to power real-time use cases such as multiplayer, chat, data synchronization, data broadcast, and notifications for highly scalable web and mobile applications around the world. Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+Using Ably's Datadog Integration, you can:
+- Use [Ably statistics][2] alongside other key metrics in Datadog
+- Correlate Ably message, channel, and connection usage for collaborative analysis in Dashboards
+- Identify unexpected activity and troubleshoot potential issues with your applications powered by Ably
```

Comment:
The overview section should include:

What does Ably do in 1-2 sentences?
What will customers get from this integration?
What types of data do you send in (and pull out if relevant)?

Can you please update the overview to reflect the above?

Reply:
> Can you please update the overview to reflect the above?

I'm unsure what to add to meet this requirement, can you give an example? So far we have:


> What does Ably do in 1-2 sentences?

The [Ably][1] platform is used to power real-time use cases such as multiplayer, chat, data synchronization, data broadcast, and notifications for highly scalable web and mobile applications around the world. Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.

> What will customers get from this integration?

I've added:
The Ably Datadog Integration sends [Ably statistics][2] metrics directly to your Datadog account.

Then we have:
Using Ably's Datadog Integration, you can:
- Use [Ably statistics][2] alongside other key metrics in Datadog
- Correlate Ably message, channel, and connection usage for collaborative analysis in Dashboards
- Identify unexpected activity and troubleshoot potential issues with your applications powered by Ably

> What types of data do you send in (and pull out if relevant)?

I think this is provided in the Data Collected/Metrics sections.